R/utils-shared.R

```diff
@@ -175,10 +175,21 @@ formatComments <- function(comments){
 ############ used in various places ############ 
 
 isEmpty <- function(val){
-  if(!is.null(val) && !is.na(val)){
-    result <- FALSE
+  if(is.null(val) || is.na(val)){
+    result <- TRUE
   } else {
+    result <- FALSE
+  }
+  return(result)
+}
+
+############ used in various places ############ 
+
+isEmptyOrBlank <- function(val){
+  if(is.null(val) || val==""){
```

Comment:
no, just null.  If it needs to handle all three cases, I can add that.  I was just going by the informal-ish requirements in slack.